plotly/tools.py

```diff
@@ -5621,4 +5618,3 @@ def make_table_annotations(self):
                         font=dict(color=font_color),
                         showarrow=False))
         return annotations
-
```

Comment:
Looks like it 😺. @Kully mind adding the newline at the end of `plotly/tools.py` back?